### PR TITLE
Fix instr when cloning from scratch

### PR DESCRIPTION
**Pull request by mratsim**
To merge in master as the build instructions for testnets don't work

```
make
make
```

is needed instead of

```
make
make update deps
```

Also now it doesn't error after the first make.

Instead this is the output (once the "build" error is fixed)

DeepinScreenshot_select-area_20191203175635

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ Once the [prerequisites](#prerequisites) are installed you can connect to testne
 git clone https://github.com/status-im/nim-beacon-chain
 cd nim-beacon-chain
 make                 # This invocation will bootstrap the build system with additional Makefiles
-make update deps     # This will build Nim and all other dependencies
+make                 # This will build Nim and all other dependencies
```
Comment (stefantalpalaru) [inline README.md]: ...and everything in TOOLS

File: Makefile [hunk 2]
```diff
@@ -29,8 +29,9 @@ build-system-checks:
 	@[[ -e "$(BUILD_SYSTEM_DIR)/makefiles" ]] || { \
 		echo -e "'$(BUILD_SYSTEM_DIR)/makefiles' not found. Running '$(GIT_SUBMODULE_UPDATE)'.\n"; \
 		$(GIT_SUBMODULE_UPDATE); \
-		echo -e "\nYou can now run '$(MAKE)' again."; \
-		exit 1; \
+		echo -e "\n✔️✔️✔️ Successfully fetched all required internal dependencies."; \
+		echo -e "        You should now \e[4mre-run '$(MAKE)' to build Nimbus\e[21m"; \
+		exit 0; \
```
Comment (stefantalpalaru) [inline Makefile]: This needs to exit with 1, in order to prevent the execution of those other dependencies of the `all` target.

**Comment by stefantalpalaru** [comment]
> the build instructions for testnets don't work

Why not? What's the error message when you run `make; make update deps`?

**Comment by mratsim** [comment]
DeepinScreenshot_select-area_20191203183820

File: README.md [hunk 3]
```diff
@@ -98,8 +98,17 @@ Once the [prerequisites](#prerequisites) are installed you can connect to testne
 git clone https://github.com/status-im/nim-beacon-chain
 cd nim-beacon-chain
 make                 # This invocation will bootstrap the build system with additional Makefiles
-make update deps     # This will build Nim and all other dependencies
+make testnet0        # This will build Nimbus and all other dependencies
+                     # and connect you to testnet0
+```
+
+If you want to connect to testnet0 without rebuilding you can use the following command
```
Comment (stefantalpalaru) [inline README.md]: False info. You don't get any "without rebuilding" from that wrapper.

**Pull request merged**